examples/all-clusters-app/all-clusters-common/src/concentration-measurement-instances.cpp

```diff
@@ -0,0 +1,239 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app/clusters/concentration-measurement-server/concentration-measurement-server.h>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::ConcentrationMeasurement;
+
+static const BitMask<Feature, uint32_t> gConcentrationMeasurementAllFeatures(Feature::kNumericMeasurement,
+                                                                             Feature::kLevelIndication, Feature::kMediumLevel,
+                                                                             Feature::kCriticalLevel, Feature::kPeakMeasurement,
+                                                                             Feature::kAverageMeasurement);
+
+static Instance * gCarbonDioxideConcentrationMeasurementServer                 = nullptr;
+static Instance * gCarbonMonoxideConcentrationMeasurementServer                = nullptr;
+static Instance * gNitrogenDioxideConcentrationMeasurementServer               = nullptr;
+static Instance * gPm1ConcentrationMeasurementServer                           = nullptr;
+static Instance * gPm10ConcentrationMeasurementServer                          = nullptr;
+static Instance * gPm25ConcentrationMeasurementServer                          = nullptr;
+static Instance * gRadonConcentrationMeasurementServer                         = nullptr;
+static Instance * gTotalVolatileOrganicCompoundsConcentrationMeasurementServer = nullptr;
+static Instance * gOzoneConcentrationMeasurementServer                         = nullptr;
+static Instance * gFormaldehydeConcentrationMeasurementServer                  = nullptr;
+
+void emberAfCarbonDioxideConcentrationMeasurementClusterInitCallback(chip::EndpointId endpoint)
```

Comment:
```suggestion
void emberAfCarbonDioxideConcentrationMeasurementClusterInitCallback(EndpointId endpoint)
```
and similar elsewhere.